dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/webdomain/EndDate.java

```diff
@@ -40,14 +43,12 @@
  * search including start and end dates. startDate=2020-10-10&endDate=2020-10-12 will include
  * anything between 2020-10-10T00:00:00.000 and 2020-10-12T23:59:59.999.
  */
-@RequiredArgsConstructor
+@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
+@Getter
```

Comment:
`Date` is a mutable type. Should we return a copy of the `date` to make this class immutable?